Makefile

```diff
@@ -112,6 +112,14 @@ $(RED_GOALS): red_%: legato_%
 	mksys -t $* $(MKSYS_ARGS_COMMON) red.sdef
 
 $(YELLOW_GOALS): yellow_%: legato_%
+	@if ! [ -e "$(BSEC_DIR)" ] ; then \
+		echo "*" ; \
+		echo "* WARNING: Bosch bsec library not found at BSEC_DIR ($(BSEC_DIR))." ; \
+		echo "*          See components/boschBsec/README.md ." >&2 ; \
+		echo "*" ; \
+		echo "Hit enter to continue." >&2 ; \
```

Comment:
What's the benefit of making the user press enter and then continuing the build?  I think we should just exit with an error.

Reply:
I did that because if the `environment` app isn't being built, then the user might actually want to continue building (because `BSEC_DIR` isn't needed in that case).  Gross, I know.
I wonder if we should instead do an `externalBuild` script in the component that checks this and prints help.

Reply:
I think it's fair to expect the user to hack the Makefile if they are hacking the yellow.sdef.  What I mean is that if they disable the BSEC related apps, they can also disable the check in the Makefile.